pages/settings/settings.html

```diff
@@ -30,7 +30,7 @@ <h3>Default folder</h3>
       <p>All future livemarks will be saved here by default</p>
       <select name="defaultFolder"></select>
       <!-- Disable prefix feature until there's decent auto updating of the prefixes -->
-      <div hidden>
+      <div>
         <h3>Bookmark prefixes</h3>
         <p>The supplied strings will be prefixed to bookmarks to indicate status. Note that titles won't update until the next feed refresh.</p>
         <label>Unread:</label>
```

Comment:
We should probably also change the name of the setting, otherwise existing users won't get this feature automatically.